portal/settings.py

```diff
@@ -11,12 +11,20 @@
 """
 
 import os
+import environ
```

Comment:
You might have missed adding the dependency for `django-environ`, and it should be added to requirements.txt